src/main/cpp/checkpoint/CheckPoint.h

```diff
@@ -59,13 +64,31 @@ class CheckPoint
 	/// Search for the last date written.
 	boost::gregorian::date GetLastDate();
 
+	/// Writes the Atlas
+	void WriteAtlas(const Atlas&);
```

Comment:
No, I mean *actually* name the parameters. Like `void LoadDisease(const std::string& filename);`.